chatmeleon-server/middlewares/error-handler.ts

```diff
@@ -0,0 +1,37 @@
+import { Request, Response, NextFunction } from 'express';
```

Comment:
@WataNekko, does this need comments? It is a bit hard to see what is going on or what its purposes are